db/migrate/20220720124734_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
+  end
```

Comment:
- [ ] As requested by the previous reviewer, this issue remains unsolved and I still run into problems running your migrations. Please see the error below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/180232963-afa35546-e57e-4b88-9a80-ff9a3056132c.png)

Reply:
@codepantha could you please kindly assist me with how to fix this? It has been working on all my previous reviews.
This relation was created in the first week of Rails and has been working fine. I am not sure how I can fix it.

Reply:
I will contact you on slack shortly then we can have a short zoom call

Reply:
Hie @codepantha thank you for the help. I have fixed the error, hopefully, it works on your end now!

Reply:
That's great. You can go ahead and ask for another review 👍